src/menu_options.c

```diff
@@ -1257,6 +1299,24 @@ setting settsystem_arr[] = {
 	ADDSET_CUSTOM("Fullscreen", FullScreenRead, FullScreenToggle, "Toggle between fullscreen and windowed mode."),
 	ADDSET_ACTION("Apply Changes", VideoApplySettings, "Restarts the renderer and applies the selected resolution."),
 
+#ifdef EZ_MULTIPLE_RENDERERS
+	ADDSET_SEPARATOR("Renderer"),
+	ADDSET_ENUM("Mode", vid_renderer, vid_renderer_enum),
+	ADDSET_ACTION("Apply Changes", RendererRestart, "Restarts the renderer."),
+#endif
```

Comment:
Not really happy with having this in a separate section with apply, but it doesn't really fit into the yes/no in the video settings above and I couldn't think of any better place.